### ISSUE TITLE
Unable to give karma: "Testcase_feedback : Invalid testcase names specified: QA:Testcase build from distgit with mock, QA:Testcase clone distgit repository, QA:Testcase download upstream sources, QA:Testcase scratch build from distgit" ..

### ISSUE THREAD

**Issue by sanjayankur31** [fedora-infra/bodhi#4099]
This was reported by a community member on AskFedora and I was able to reproduce it:
https://ask.fedoraproject.org/t/how-to-provide-feedback-to-testing-updates-in-bodhi/8643

When giving karma, we're seeing errors:

```
Testcase_feedback : Invalid testcase names specified: QA:Testcase build from distgit with mock, QA:Testcase clone distgit repository, QA:Testcase download upstream sources, QA:Testcase scratch build from distgit
```

and it won't register the karma at all. Here's the offending update:

https://bodhi.fedoraproject.org/updates/FEDORA-2020-b81268fd86

The test cases look OK at a glance.

**Comment by laolux** [comment]
Hi everyone, as mentioned above, I originally inquired about this on AskFedora. The problem still persists for me, but apparently others are able to give good karma. This is quite troublesome, as the package is gaining positive karma and might soon be pushed to stable even though I am continuing to experience problems with this update.

**Comment by sanjayankur31** [comment]
Duplicate of https://github.com/fedora-infra/bodhi/issues/4088  

Closing.

**Comment by laolux** [comment]
I just saw a similar bug, but the error message is a tiny bit different: #4088 
However, that one is supposed to be fixed with #4090 , so either this bug here is different, or the fix is not used (yet) by fedora.
